application/models/models.go

```diff
@@ -40,6 +40,13 @@ const (
 	PermissionDelete
 	PermissionDenied
 
+	ItemStatusChangeSubmitted    = "Submitted for approval"
+	ItemStatusChangeAutoApproved = "Auto approved"
+	ItemStatusChangeApproved     = "Approved by "
+	ItemStatusChangeRevisions    = "Revisions requested by "
+	ItemStatusChangeDenied       = "Denied by "
+	ItemStatusChangeInactivated  = "Inactivated by "
```

Comment:
```suggestion
	ItemStatusChangeInactivated  = "Deactivated by "
```
https://english.stackexchange.com/questions/312339/inactive-or-deactivated